### PR TITLE
Translate wiki followup

### PR DESCRIPTION
**Pull request by Tolu-Mals**
# Summary

In this pr I
- updated the navbar to not show flags
- updated the wiki translation prompts for better translations
- fixed a small issue with translate btn
- add trigger for GA event when translation button is clicked

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/2247
closes https://github.com/EveripediaNetwork/issues/issues/2259

### CODE REVIEW

**Review by Simplypheyie96 — APPROVED** [review]

File: src/components/Wiki/WikiPage/WikiMainContent.tsx [hunk 1]
```diff
@@ -128,6 +128,13 @@ const WikiMainContent = ({ wiki: wikiData }: WikiMainContentProps) => {
     )
 
     const handleClick = async () => {
+      gtag('event', 'translate_btn_click', {
```
Comment (kesar) [inline src/components/Wiki/WikiPage/WikiMainContent.tsx:131]: logEvent

Comment (kesar) [reply src/components/Wiki/WikiPage/WikiMainContent.tsx:131]: logEvent({
        action: 'TRANSLATE_WIKI',
        category: btnLocale,
        label: wikiData.id,
        value: 1,
      })

File: src/pages/api/translate-wiki.ts [hunk 2]
```diff
@@ -16,13 +16,28 @@ export default async function handler(
...
         messages: [
           {
             role: 'system',
-            content: `You will be provided with a peice of markdown text in English. Your task is to translate it into Korean, preserving the formatting. Note the following:
-              1. Any peice of text surrounded by double asterisks should be immediately fellowed by a space. 
-              For example: **ABC**D becomes **ABC** D .Where A, B, C are letters in Korean.
-              Example 2: **리플 (Ripple)**은 becomes **리플 (Ripple)** 은 
+            content: `You will be provided with a peice of markdown text in English. Your task is to output the text in Korean, preserving the formatting. While outputing the translation, make sure to do the following:
+              1. Do not translate names to Korean, output names the way they are in the original english text
+              1. Any peice of characters that start and end with double asterisks(**) should be immediately fellowed by an empty space( ).
```
Comment (Adebesin-Cell) [inline src/pages/api/translate-wiki.ts:25]: ``piece``

File: src/components/Layout/Navbar/DesktopNav.tsx [hunk 3]
```diff
@@ -60,8 +57,10 @@ const DesktopNav = () => {
             },
           }}
           rightIcon={<ChevronDownIcon color="linkColor" />}
+          iconSpacing={1}
+          defaultValue={lang}
         >
-          {langFlagMap[lang]}
+          {lang.toUpperCase()}
```
Comment (Adebesin-Cell) [inline src/components/Layout/Navbar/DesktopNav.tsx:63]: Use css capitalize.

File: src/pages/api/translate-wiki.ts (hunk 2 above)
Comment (Tolu-Mals) [reply src/pages/api/translate-wiki.ts:25]: Nice catch

**Review by Softdev1 — APPROVED** [review]

**Pull request merged**